### ISSUE TITLE
A11y_AzureToolsStorageExplorer_BlobOperations_Rename_BlobContainer_ScreenReader:Voiceover does not announce the status message "New name provided is identical"

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/AzureStorageExplorer#2612]
“[Check out Accessibility Insights](https://accessibilityinsights.io/)! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yWCAG2.1; #A11YTCS; #DesktopApp;  #WCAG4.1.3; #VoiceOver; #Mac; #Benchmark; #A11ySev2; #Azure Tools Storage Explorer;

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.12.0
MacOS: Catalina 10.15.3

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once created, navigate to the created blob container and right click on it.
6. Select 'clone blob with new name' put the same name of the blob which we have to clone.
7. Click on Clone button
8. Observe what voiceover announces

**Actual:**
Voiceover does not announce the status message "New name provided is identical" which is appearing in Activities Tab as soon as hit enter on Clone button

**Expected:**
Voiceover should announces the Status message after cloning with new name which is appearing in Activities Tab

**User Impact:**
Screen reader users won't get information/message about the action performed on "Clone' control.

**Comment by Shivamd1** [comment]
We have uploaded repro video of this bug in [OneDrive](https://microsoft-my.sharepoint.com/:f:/p/v-subha/EvxHdg1zkN9PknF1uOxnOiQBejpxKYg9n21DydgQ_om0ew?e=WK2Zih) link with name "2612_A11y_AzureToolsStorageExplorer_BlobOperations_Rename_BlobContainer_ScreenReader" please refer the video for reference.
Please let us know if there is any problem in accessing the link

**Comment by JasonYeMSFT** [comment]
@Shivamd1 This seems to be already fixed. We now display an error dialog and the content will be announced when the dialog prompts.

**Comment by Shivamd1** [comment]
@MayuriNaik Can you please look into it?

**Comment by Ashutoshsahoo164** [comment]
GitHubTag: #BM-Azure Tools Storage Explorer-MAC-Aug2020

**Comment by BhanuPrakashGit** [comment]
Verified this bug and issue is not repro'ing.
**GithubTags:**#Closed;
PFA for reference
[2612_Closed.zip](https://github.com/microsoft/AzureStorageExplorer/files/10753485/2612_Closed.zip)

**Comment by InduPriya1805** [comment]
GithubTags:#BM-Azure Tools Storage Explorer-MAC-Aug2020;
